napalm/eos/eos.py

```diff
@@ -57,6 +57,7 @@ class EOSDriver(NetworkDriver):
     HEREDOC_COMMANDS = [
         ("banner login", 1),
         ("banner motd", 1),
+        ("comment", 1),
```

Comment:
I didn't follow how this was used relative to _mode_comment_convert?

Reply:
It wasn't, but it does reflect that the following is valid EOS config/CLI commands:

```
ip access-list standard example
    comment
    This is a multi-line comment
    HEREDOC command
    EOF
    permit host 192.0.2.1
```

 It gets displayed in the running-config as 

```
ip access-list standard example
    !! This is a multi-line comment
    !! HEREDOC command
    permit host 192.0.2.1
```